### ISSUE TITLE
Can't get simulcast to work.

### ISSUE THREAD

**Issue by lutsykigor** [oney/react-native-webrtc#237]
I am new to native WebRTC, I am trying to use simulcast on Android.
Unfortunately, I can't get it to work, does react-native-webrtc support simulcast?

I am using this lib https://github.com/jitsi/sdp-simulcast to enable simulcast in SDP.
Simulcast is supported in Chrome browser, but I am not sure that it is part of native WebRTC stack, or it is something internal to Chrome.

Here is my SDP:
=====================================================
v=0
o=- 814997227879783433 5 IN IP4 127.0.0.1
s=-
t=0 0
a=msid-semantic: WMS 0836cc8e-a7bb-47e9-affb-0599414bc56d
a=group:BUNDLE audio video data
m=audio 54405 RTP/SAVPF 111 103 104 126
c=IN IP4 172.29.32.39
a=rtpmap:111 opus/48000/2
a=rtpmap:103 ISAC/16000
a=rtpmap:104 ISAC/32000
a=rtpmap:126 telephone-event/8000
a=fmtp:111 minptime=10;useinbandfec=1
a=rtcp:9 IN IP4 0.0.0.0
a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
a=extmap:3 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=setup:passive
a=mid:audio
a=sendrecv
a=ice-ufrag:adPg
a=ice-pwd:Xsr05Mq8S7CR44DAnusZE26F
a=fingerprint:sha-256 6A:39:DE:11:24:AD:2E:4E:63:D6:69:D3:85:05:53:C7:3C:38:A4:B7:91:74:C0:91:44:FC:94:63:7F:01:AB:A9
a=candidate:1581043602 1 udp 2122260223 172.29.32.39 54405 typ host generation 0
a=ssrc:124723944 cname:peDGrDD6WsxUOki/
a=ssrc:124723944 msid:dcbb0236-cea5-402e-9e9a-595c65ffcc2a 40abf2d3-a415-4c68-8c17-2a038e8bebcf
a=ssrc:124723944 mslabel:dcbb0236-cea5-402e-9e9a-595c65ffcc2a
a=ssrc:124723944 label:40abf2d3-a415-4c68-8c17-2a038e8bebcf
a=rtcp-mux
m=video 9 RTP/SAVPF 100
c=IN IP4 0.0.0.0
a=rtpmap:100 VP8/90000
a=rtcp:9 IN IP4 0.0.0.0
a=rtcp-fb:100 ccm fir
a=rtcp-fb:100 nack
a=rtcp-fb:100 nack pli
a=rtcp-fb:100 goog-remb
a=extmap:3 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=setup:passive
a=mid:video
a=sendrecv
a=ice-ufrag:adPg
a=ice-pwd:Xsr05Mq8S7CR44DAnusZE26F
a=fingerprint:sha-256 6A:39:DE:11:24:AD:2E:4E:63:D6:69:D3:85:05:53:C7:3C:38:A4:B7:91:74:C0:91:44:FC:94:63:7F:01:AB:A9
a=ssrc:1757014965 msid:0836cc8e-a7bb-47e9-affb-0599414bc56d bdbd2c0a-7959-4578-8db5-9a6a1aec4ecf
a=ssrc:1757014965 cname:peDGrDD6WsxUOki/
a=ssrc:1479742055 msid:0836cc8e-a7bb-47e9-affb-0599414bc56d bdbd2c0a-7959-4578-8db5-9a6a1aec4ecf
a=ssrc:1479742055 cname:peDGrDD6WsxUOki/
a=ssrc:1089111804 msid:0836cc8e-a7bb-47e9-affb-0599414bc56d bdbd2c0a-7959-4578-8db5-9a6a1aec4ecf
a=ssrc:1089111804 cname:peDGrDD6WsxUOki/
a=ssrc-group:SIM 1757014965 1479742055 1089111804
a=rtcp-mux
m=application 9 DTLS/SCTP 5000
c=IN IP4 0.0.0.0
b=AS:30
a=setup:passive
a=mid:data
a=ice-ufrag:adPg
a=ice-pwd:Xsr05Mq8S7CR44DAnusZE26F
a=fingerprint:sha-256 6A:39:DE:11:24:AD:2E:4E:63:D6:69:D3:85:05:53:C7:3C:38:A4:B7:91:74:C0:91:44:FC:94:63:7F:01:AB:A9
a=sctpmap:5000 webrtc-datachannel 1024
====================================================

Any advice appreciated.

**Comment by saghul** [comment]
@lutsykigor You need to polyfill `window.chrome`, because that's how sdp-simulcast detects support (at the moment) https://github.com/jitsi/sdp-simulcast/blob/master/lib/index.js#L427

**Comment by lutsykigor** [comment]
@saghul thanks for answer, maybe it is offtopic, but does Jitsi Meet mobile native client polyfills  `window.chrome`, and supports simulcast?

**Comment by saghul** [comment]
@lutsykigor Nope, we don't currently do that. I haven't had a chance to try it out, but I;m not sure how beneficial it would be. Simulcast is good for the receiver, at the cost of some extra effort on the senser's side, but given it's mobiles we're talking about, the extra effort might get translated into lower battery life, which is not a good thing.

Codecs with spatial scalability such as VP9 might help since only one encoding is made but AFAIK VP9 is quite CPU intensive. OTOH, H264 is usually hardware accelerated, so that mifght be a better option.

I haven't done tests or meassurements yet, to take all of the above with a kilo of salt :-)
